src/Stratis.Bitcoin.Features.Apps/AppsFeature.cs

```diff
@@ -56,12 +61,23 @@ public static IFullNodeBuilder UseApps(this IFullNodeBuilder fullNodeBuilder)
                     .AddFeature<AppsFeature>()
                     .FeatureServices(services =>
                     {
+                        TryEnsureAppsFolderExists(services);
```

Comment:
I don't think this is the right place to do this, it should go in the Feature `Initialize` method